tests/e2e/performance/test_pvc_clone_performance.py

```diff
@@ -79,14 +50,60 @@ def base_setup(self, interface_type, pvc_size, pvc_factory, pod_factory):
             status=constants.STATUS_RUNNING
         )
 
-    def test_clone_create_delete_performance(self, interface_type, pvc_size, file_size, teardown_factory):
+    @skipif_ocs_version('<4.6')
+    @pytest.mark.parametrize(
+        argnames=["interface_type", "pvc_size", "file_size"],
+        argvalues=[
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL, '1', '600Mi'], marks=pytest.mark.polarion_id('OCS-2356')
+            ),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL, '25', '15Gi'], marks=pytest.mark.polarion_id('OCS-2340')
+            ),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL, '50', '30Gi'], marks=pytest.mark.polarion_id('OCS-2357')
+            ),
+            pytest.param(
+                *[constants.CEPHBLOCKPOOL, '100', '60Gi'], marks=pytest.mark.polarion_id('OCS-2358')
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM, '1', '600Mi'], marks=pytest.mark.polarion_id('OCS-2341')
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM, '25', '15Gi'], marks=pytest.mark.polarion_id('OCS-2355')
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM, '50', '30Gi'], marks=pytest.mark.polarion_id('OCS-2359')
+            ),
+            pytest.param(
+                *[constants.CEPHFILESYSTEM, '100', '60Gi'], marks=pytest.mark.polarion_id('OCS-2360')
+            )
+        ]
+    )
+    @pytest.mark.usefixtures(base_setup_single_clone.__name__)
```

Comment:
suggest following the convention of passing the fixture name to the test function:

```python
def test_single_clone_create_delete_performance(self, interface_type, pvc_size, file_size, teardown_factory, base_setup_single_clone)
```